lib/fs.js

```diff
@@ -376,6 +376,12 @@ fs.exists = function(path, callback) {
   }
 };
 
+Object.defineProperty(fs.exists, internalUtil.promisify.custom, {
+  enumerable: false,
+  value: (path) => new Promise((resolve) => fs.exists(path, resolve))
```

Comment:
No, the point of promisify is to give users a standard interface for turning Node-style callbacks into functions that return Promises. And since this is a deprecated API, I don’t think there’s a need to micro-optimize.